modules/installation-mirror-repository.adoc

```diff
@@ -48,13 +87,51 @@ the pull secret for your mirror registry that you created.
 <6> The release mirror. For a production release, you must specify
 `ocp-release`.
 
+. Build the signature for the content that you plan to mirror and verify that it
+is signed by an official Red Hat key:
+.. Set the digest for the version to install to:
++
+----
+$ DIGEST="$(oc adm release info quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE} | sed -n 's/Pull From: //p')" <1>
+----
+.. Build a signature URI:
++
+----
+$ URI="https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/${DIGEST/:/=}/<signature_name>" <1>
+----
+<1> Specify the name of the signature to store.
+
+.. Download the signature:
++
+----
+$ wget "${URI}"
+----
+
+.. Verify the signature:
++
+----
+$ gpg --verify <signature_name> <1>
```

Comment:
When I run this command, I get this:
`
$ gpg --verify signature-1
gpg: Signature made Tue 08 Oct 2019 10:12:39 AM EDT using RSA key ID F21541EB
gpg: Can't check signature: No public key
`
Seem like here miss some steps.

Reply:
Ah, prob need to import at least beta2 from the release key page.

Reply:
https://access.redhat.com/security/team/key links to https://access.redhat.com/articles/3530471 as "a manual process has been established".  Maybe just lean on that for downloading a key and verifying with that?  For releases, the signatures are so far all by the beta2 key, which is https://www.redhat.com/security/data/f21541eb.txt .

Reply:
I try to follow the procedure mentioned in the doc, but still failed, any idea?
`
$ wget https://www.redhat.com/security/data/f21541eb.txt
$ gpg --no-default-keyring --keyring ./temp.keyring --import ./f21541eb.txt
gpg: /root/.gnupg/trustdb.gpg: trustdb created
gpg: key F21541EB: public key "Red Hat, Inc. (beta key 2) <security@redhat.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1  (RSA: 1)
$ gpg --verify signature-1
gpg: Signature made Tue 08 Oct 2019 10:12:39 AM EDT using RSA key ID F21541EB
gpg: Can't check signature: No public key
`

Reply:
@wking, will you PTAL?

cc: @abhinavdahiya since Trevor's not working on install work this sprint

Reply:
@wking @abhinavdahiya *bump*.

Reply:
Looks like the issue with @jianlinliu's earlier attempt was 4.2.2's signature-1 coming from release 2 instead of beta 2:

```console
$ DIGEST="$(oc adm release info quay.io/openshift-release-dev/ocp-release:4.2.2 | sed -n 's/Pull From: .*@//p')"
$ echo "${DIGEST}"
sha256:dc782b44cac3d59101904cc5da2b9d8bdb90e55a07814df50ea7a13071b0f5f0
$ URI="https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/${DIGEST/:/=}/signature-1"
$ echo "${URI}"
https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/sha256=dc782b44cac3d59101904cc5da2b9d8bdb90e55a07814df50ea7a13071b0f5f0/signature-1
$ wget https://www.redhat.com/security/data/fd431d51.txt
$ gpg --no-default-keyring --keyring ./temp.keyring --import fd431d51.txt
$ gpg --no-default-keyring --keyring ./temp.keyring --verify signature-1
gpg: Signature made Fri 25 Oct 2019 08:47:11 AM PDT using RSA key ID FD431D51
gpg: Good signature from "Red Hat, Inc. (release key 2) <security@redhat.com>"
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 567E 347A D004 4ADE 55BA  8A5F 199E 2F91 FD43 1D51
```

Reply:
So if we're going to document which keys to import, probably best to say "both f21541eb and fd431d51" to cover our bases.

Reply:
@kalexand-rh Great. @wking's steps works well now. pls update those steps in our doc.